### ISSUE TITLE
assert has been happened when the software was closed

### ISSUE THREAD

**Issue by funny1624** [cnr-isti-vclab/meshlab#1183]
assert has been happened when the software was closed.' "group->context() == q_ptr" in file qgl.cpp, line 1729 '

![image](https://user-images.githubusercontent.com/19150696/147914954-106b2d7a-6331-4ac8-a08a-4edcb4def439.png)

**Comment by alemuntoni** [comment]
I am aware of this issue that happens only on windows in debug mode. 
It will be automatically removed when meshlab will switch to Qt6.

**Comment by josephshen** [comment]
> I am aware of this issue that happens only on windows in debug mode. It will be automatically removed when meshlab will switch to Qt6.

Hi Alessandro, when will meshlab switch to Qt6, did our meshlab dev group have a confirmed time?

**Comment by alemuntoni** [comment]
@josephshen, nope, sorry. I am struggling on solving a high number of issues about the migration from QGL to QOpenGL classes. I just don't know when we will be able to switch to Qt6.
